### PR TITLE
seaverbranch-models

### PR DESCRIPTION
**Pull request by seaverstanley**
## Proposed changes
finally added models
## Steps To Test
_Please include expected output._

## What kind of change does this PR introduce?
Please check one of the following:
- [ ] Bug fix
- [ ] Documentation update
- [ ] New feature
- [ X] Other

### CODE REVIEW

File: BangazonAPI/Models/Order.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        public int Id { get; set; }
+
+        public int PaymentTypeId { get; set; }
+
+        public int CustomerId { get; set; }
+
+        public List<Product> listOfProducts { get; set; } = new List<Product>();
+
+        public List<Customer> listOfCustomer { get; set; } = new List<Customer>();
```
Comment (Tommy-Spurlock) [inline BangazonAPI/Models/Order.cs]: Orders class does not need a list of customers.

File: BangazonAPI/Models/TrainingProgram.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    public class TrainingProgram
+    {
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        public int StartDate { get; set; }
+
+        public int EndDate { get; set; }
+    }
```
Comment (Tommy-Spurlock) [inline BangazonAPI/Models/TrainingProgram.cs:19]: Employees who signed up for a training program should be included in the response. Needs all the columns on the ERD.

**Review by Tommy-Spurlock — CHANGES_REQUESTED** [review]
Some models need to be altered. Each model should have all the columns from its perspective table in the database. Also, make sure to check the tickets for any list that may be needed in a model.

File: BangazonAPI/Models/TrainingProgram.cs (hunk 2 above)
Comment (seaverstanley) [reply BangazonAPI/Models/TrainingProgram.cs:19]: I added a list of employees to this class in the new updated code :)

File: BangazonAPI/Models/Order.cs (hunk 1 above)
Comment (seaverstanley) [reply BangazonAPI/Models/Order.cs]: I got rid of the list of customers in the updated code :)

**Comment by ColonelSped** [comment]
https://github.com/Obedient-Lobsters/BangazonAPI/pull/57
@seaverstanley try using this as a good example for your PR document. Upon edit, please request another review.

**Comment by ColonelSped** [comment]
👍

**Comment by jessicaayerst** [comment]
Looks good. All models are there.

**Comment by kelli-evans09** [comment]
Works great on mine! I can view all models with correct info in each class :)

**Pull request merged**